packages/atomic/src/components/result-template-components-v1/atomic-result-icon/atomic-result-icon.tsx

```diff
@@ -11,7 +11,7 @@ import {fileTypeIcons} from './file-type-icons';
 @Component({
   tag: 'atomic-result-icon-v1',
   styleUrl: 'atomic-result-icon.pcss',
-  shadow: false,
+  shadow: true,
```

Comment:
`atomic-result-icon` can now use a shadow DOM. In this case, I needed it to render the `slot` conditionally.